monitoring/README.md

```diff
@@ -1,112 +1,66 @@
-# Monitoring Tools for KubeStellar (Prometheus, Grafana and Node Exporter)
+# Monitoring Tools for KubeStellar (Prometheus, Pyroscope & Grafana)
 
-<img src="images/kubestellar-architecture-aws.png" width="200%" height="200%" title="kubestellar-architecture-aws">
+<img src="images/ks-monitoring-setup.png" width="200%" height="200%" title="kubestellar-monitoring-setup">
 
 
 ### Description
-This example shows how to deploy monitoring tools (Prometheus, Grafana and node exporter) for KubeStellar components (core and edge regions) - see architecture image above. Prometheus server is deployed in the core region running the KCP server alongside the components for KCP edge. A Prometheus agent is deployed in the edge regions running the edge workload execution cluster (wec)
+This example shows how to deploy monitoring tools (Prometheus, Grafana and Pyroscope) for KubeStellar control plane components - see architecture image above.
 
-1. Create your hosts file with the list of target hosts (KCP server & Edge Workload Execution Clusters (WEC))
+1. Install the monitoring tools:
 
+```bash
+$ ./install-ks-monitoring.sh
```

Comment:
This seems to be making an assumption about what the user's current working directory is.

Reply:
Fixed!